Controller/Admin/TransactionList.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Wirecard\Oxid\Controller\Admin;
+
+use Wirecard\Oxid\Model\Transaction;
+
+use OxidEsales\Eshop\Core\Registry;
+use OxidEsales\Eshop\Application\Controller\Admin\AdminListController;
+
+/**
+ * Controls the transaction list view.
+ */
+class TransactionList extends AdminListController
+{
+    protected $_sListClass = Transaction::class;
```

Comment:
Let's please add an **"@Override" PHPDoc** comment (not officially existing, ignored) indicating an overridden variable. Currently, overridden variables are not obvious to other devs.

Reply:
Same applies to functions we are overriding.

Reply:
Isn't that what `@inheritdoc` is used for?